nixos/modules/services/misc/paperless.nix

```diff
@@ -199,9 +207,41 @@ in
       defaultText = literalExpression "pkgs.paperless-ngx";
       description = lib.mdDoc "The Paperless package to use.";
     };
+
+    backup = {
+
+      enable = mkEnableOption (lib.mdDoc "Paperless backups");
+
+      directory = mkOption {
+        type = types.str;
+        default = cfg.dataDir + "/backup";
```

Comment:
For this to work (also noted by one of the CI scripts), you also need to set a `defaultText` attribute

Reply:
Ack.